tests/unit_tests/blas/extensions/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@
 #===============================================================================
 
 # Build object from all test sources
-set(EXTENSIONS_SOURCES "gemm_ext.cpp" "gemm_ext_off.cpp" "gemmt.cpp" "gemmt_usm.cpp")
+set(EXTENSIONS_SOURCES "gemmt.cpp" "gemmt_usm.cpp")
```

Comment:
`gemm_bias` seems to be missing?

Reply:
Thanks for the feedback. gemm_bias test has been added.